tla-io/src/main/resources/issueTemplate.md

```diff
@@ -0,0 +1,30 @@
+<!-- Thank you for filing a report! Please ensure you have filled out all -->
+<!-- sections, as it help us to address the problem effectively. -->
+
+## Input specification
+
+```
+__SPEC
+```
+
+## The command line parameters used to run the tool
+
+```
```

Comment:
Any added value to making this a standalone file that has to be included in the resources that you have in mind?

My initial thought is that this adds some unneeded complexity (an additional resource, then adhoc templating logic), but it seems our usecase could be served just as well with a function that has this string in and just uses Scala's string interpolation for templating. What do you think?

Reply:
You could just have a string, that's true, but the string would be messy to write, and it's easy to edit the `.md` file as a standalone file to get the desired formatting. I guess string interpolation is better than replacement, so the question is really whether we intend to change the bug report template at all. If the answer is "not really", I suppose we can copy the file into a string.

Reply:
I'm ambivalent on this, if you'd prefer a string it's easy enough to include.